src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,35 @@
+package tasks;
+
+import java.io.Serializable;
+
+public class Task implements Serializable {
+    private String task;
+    private boolean isDone;
+    private String Symbol;
+    private final String doneSymbol = "[X]";
```

Comment:
If you intend to declare a constant, would it be better to declare it as a static final as it is a class level variable? Also, the convention for naming constants is fully uppercase letters with underscores to separate the words ([source](https://se-education.org/guides/conventions/java/intermediate.html#:~:text=Constant%20names%20must%20be%20all%20uppercase%20using%20underscore%20to%20separate%20words.)).